seatunnel-dist/pom.xml

```diff
@@ -720,6 +720,7 @@
                     <groupId>org.apache.seatunnel</groupId>
                     <artifactId>seatunnel-spark-3-starter</artifactId>
                     <version>${project.version}</version>
+                    <scope>provided</scope>
```

Comment:
I am not found the spark and flink License in `seatunnel-dist/release-docs/LICENSE`. It is necessary if you put `seatunnel-spark-3-starter` in release package.

Reply:
done